docs/reference/glossary.asciidoc

```diff
@@ -105,6 +115,19 @@
   +
   See also <<glossary-routing,routing>>
 
+[[glossary-query]] query ::
+
+  Query refers to all queries which not only determine if a document matches, 
```

Comment:
You start by contrasting queries with something else (filters I guess), but maybe its possible to start the definition in general? Maybe talk about queries as that what defines a search request and then later differentiate between queries that score and queries that don't score (filters)? Not sure about the best phrasing for this, I think the current formulations are already quiete good, maybe also just change the ordering a bit?